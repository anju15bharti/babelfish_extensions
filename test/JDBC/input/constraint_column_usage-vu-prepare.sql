create database db_constraint_column_usage;
go

Use db_constraint_column_usage;
go

create table tbl1(arg1 int, arg2 int, primary key(arg1));
go

create table tbl2(arg3 int, arg4 int, primary key(arg3), foreign key(arg4) references tbl1(arg1));
go

create schema sc1;
go

create table tbl3 (arg5 int, arg6 int, primary key (arg5,arg6));
go

create table sc1.tbl4 (arg7 int, arg8 int, check ( arg7 > 0  and arg8 < 0));
go

create table tbl5 (arg9 int, arg10 int, arg11 int, foreign key(arg10,arg11) references tbl3(arg5,arg6));
go
