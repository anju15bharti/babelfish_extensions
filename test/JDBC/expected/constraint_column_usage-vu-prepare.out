create database db1;
go

Use db1;
go

create table tbl1(a int, b int, primary key(a));
go

create table tbl2(a int, b int, primary key(a), foreign key(b) references tbl1(a));
go

create schema sc1;
go

create table tbl3 (a int, b int, primary key (a,b));
go

create table sc1.tbl4 (a int, b int, check ( a > 0  and b < 0));
go

create table tbl5 (a int, b int, c int, foreign key(b,c) references tbl3(a,b));
go

